calicoctl/calico_ctl/node.py

```diff
@@ -570,6 +570,22 @@ def _setup_ip_forwarding():
         print "ERROR: Could not enable ipv6 forwarding."
         sys.exit(1)
 
+def _set_nf_conntrack_max():
+    """
+    A common problem on Linux systems is running out of space in the conntrack
+    table, which can cause poor iptables performance. This can happen if you
+    run a lot of workloads on a given host, or if your workloads create a lot
+    of TCP connections or bidirectional UDP streams.
+
+    To avoid this becoming a problem, we recommend increasing the conntrack
+    table size. To do so, run the following commands:
+    """
+    try:
+        with open('/proc/sys/net/netfilter/nf_conntrack_max', 'w') as f:
+            f.write("1000000")
+    except:
+        print "ERROR: Could not set nf_contrack_max"
+        sys.exit(1)
```

Comment:
Do we want to fail entirely if we can't do this? Or should we just emit a warning and let it continue?

It's not like we won't work at all in this case.